src/CommitmentsV2/SFA.DAS.CommitmentsV2/Configuration/CustomisedFeaturesConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using SFA.DAS.Authorization.Features.Configuration;
+
+namespace SFA.DAS.CommitmentsV2.Configuration
+{
+    public class CustomisedFeaturesConfiguration : FeaturesConfiguration
```

Comment:
my assumption is that the email will become mandatory for all the employers/providers - after that we will need to do clean up task to remove all the code for enable email. If this is true, I would say that we keep CustomisedFeatureConfiguration as a seperate service instead of extending FeaturesConfiguration, as it may help in removing it later down the line.

Reply:
There is potential value in keeping the PrivateBetaList for future development. The pattern will show how we can implement this locally. The clean up code should potentially remove the sub optimal code we use when fetching the cohort